### PR TITLE
Add compatibility for Beta 36 (1.5)

### PR DESCRIPTION
**Pull request by mahtaran**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog of PvPMode for Minecraft 1.7.10:
 
+## 1.5.1-BETA
+### LOTR compatibility:
+* Updated to Beta 36.1 of the LOTR Mod
```
Comment (CraftedMods) [inline CHANGELOG.md:5]: v36.2

**Review by CraftedMods — CHANGES_REQUESTED** [review]
Update the LOTR version in README.md, otherwise it looks fine

**Review by CraftedMods — CHANGES_REQUESTED** [review]
You forgot the ones in README.md

**Comment by Mahtaran** [comment]
You are totally right, whoops

**Review by CraftedMods — APPROVED** [review]

**Pull request merged**